terraform-plans/templates/github/tox.ini.tftpl

```diff
@@ -70,8 +74,12 @@ commands = pytest {toxinidir}/tests/unit \
 deps =
   pytest
   pytest-cov
+%{ if root_requirements_txt == "yes" ~}
   -r {toxinidir}/requirements.txt
+%{ endif ~}
+%{ if unittest_type != "none" && unit_requirements_txt == "yes" ~}
   -r {toxinidir}/tests/unit/requirements.txt
+%{ endif ~}
```

Comment:
If there are python based unit tests, could we simply make it a convention that we always include a requirements.txt file in the project? Even if it's empty.